Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,30 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+
+// Header for the receipt
+console.log("QTY     ITEM                TOTAL");
+
+// Initialize total cost
+let totalCostPence = 0;
+
+// Iterate through each order item
+order.forEach(({ itemName, quantity, unitPricePence }) => {
+  // Calculate total price for the current item
+  const itemTotalPence = quantity * unitPricePence;
+  totalCostPence += itemTotalPence;
+
+  // Format the item's total price in pounds
+  const itemTotalPounds = (itemTotalPence / 100).toFixed(2);
+
+  // Log the item's details with proper formatting
+  console.log(
+    `${quantity.toString().padEnd(7)}${itemName.padEnd(20)}${itemTotalPounds}`
```

Comment:
Right now, your headers (ITEM, TOTAL) are one space to the right of the start of the values.

How could you change this code so that:
1. They are aligned the same
2. Even if you decided you wanted the second (item) column to move to the right one space, you would only need to change one number, and not change two different bits of code
?